plugins/confluent/credentials.go

```diff
@@ -0,0 +1,91 @@
+package confluent
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func CloudCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.Credentials,
+		DocsURL:       sdk.URL("https://docs.confluent.io/confluent-cli/current/command-reference/confluent_login.html"),
```

Comment:
Maybe [this URL](https://docs.confluent.io/cloud/current/access-management/identity/user-accounts.html#local-user-username-password) would be a better docs URL for this credential type?
```suggestion
		DocsURL:       sdk.URL("https://docs.confluent.io/cloud/current/access-management/identity/user-accounts.html#local-user-username-password"),
```